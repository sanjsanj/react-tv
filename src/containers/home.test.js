import React from 'react';
import { shallow } from 'enzyme';

import Home from './home';

it('renders correctly', () => {
  const props = {
    messages: [],
    loadMessages: () => {},
  };
  const tree = shallow(<Home.WrappedComponent {...props} />);
  expect(tree).toMatchSnapshot();
});
