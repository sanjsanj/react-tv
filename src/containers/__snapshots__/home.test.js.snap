// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should mount 1`] = `
<ul>
  <li
    key="cd445e6d-e514-424f-ba8f-16ec842002c6"
  >
    Quisque erat eros, viverra eget, congue eget, semper rutrum, nulla.
  </li>
  <li
    key="b03569ae-ccbf-4975-8040-4daba638b407"
  >
    Integer a nibh. In quis justo. Maecenas rhoncus aliquam lacus.
  </li>
</ul>
`;
